### ISSUE TITLE
Naming convention: Uppercase device codename

### ISSUE THREAD

**Issue by z3ntu** [postmarketOS/pmbootstrap#241]
Either the wiki doesn't have that infromation: https://github.com/postmarketOS/pmbootstrap/wiki/Package-naming-convention
Or the pmbootstrap tool doesn't like uppercase package names.

Example:
```
luca@g550jk /m/h/p/pmbootstrap> ./pmbootstrap.py checksum device-fairphone-FP2
CRITICAL:root:You will need to create a device-specific package
CRITICAL:root:before you can continue. Please create at least the
CRITICAL:root:following files:
CRITICAL:root:/mnt/hdd/postmarketOS/pmbootstrap/aports/device-fairphone-fp2/APKBUILD
CRITICAL:root:/mnt/hdd/postmarketOS/pmbootstrap/aports/device-fairphone-fp2/deviceinfo
Traceback (most recent call last):
  File "./pmbootstrap.py", line 69, in <module>
    sys.exit(main())
  File "./pmbootstrap.py", line 35, in main
    args = pmb.parse.arguments()
  File "/mnt/hdd/postmarketOS/pmbootstrap/pmb/parse/arguments.py", line 236, in arguments
    setattr(args, "deviceinfo", pmb.parse.deviceinfo(args))
  File "/mnt/hdd/postmarketOS/pmbootstrap/pmb/parse/deviceinfo.py", line 38, in deviceinfo
    raise RuntimeError("Incomplete device information")
RuntimeError: Incomplete device information
```

**Comment by craftyguy** [comment]
What did you name the device package folder under aports? `device-fairphone-FP2` or `device-fairphone-fp2`?

**Comment by z3ntu** [comment]
`device-fairphone-FP2`

**Comment by craftyguy** [comment]
Ah, it seem that the prompt for 'device' is converting the response to lower case by not explicitly setting `lowercase_answer=False` in the call to `pmb.helpers.cli.ask` (it defaults to `True`)

```
cfg["pmbootstrap"]["device"] = pmb.helpers.cli.ask(args, "Device",  
                                                   None, args.device)
```
This means that anything you input will be converted to lower case.

I can submit a PR to correct this, though it'll be up to @ollieparanoid whether he wants to allow case-sensitive device naming. I can think of several arguments for and against that :)

**Comment by ollieparanoid** [comment]
Thank you for making this ticket. Indeed, we were missing a specification of the casing in that wiki article.

[Alpine's packaging policy](https://wiki.alpinelinux.org/wiki/Package_policies) says:
> All package names should be lowercase.

I'd rather not diverge from that, we might even run into issues when that is the expected way by Alpine. I've also suggested in the IRC that we do it that way and no one strictly spoke out against it. So I've adjusted the wiki page to allow lowercase device names only.
